model_compiler/python/nnir.py

```diff
@@ -357,19 +357,28 @@ def updateLocals(self):
                 elif node.type in ['reshape']:
                     input = self.tensor_dict[node.inputs[0]]
                     param = node.attr.get('shape')
+                    #param = input.shape
```

Comment:
do we need this as a comment?

Reply:
This is what needs to be used for ONNX models Version 1.3. I can take it off and add it when I work on version 1.3

Reply:
Yes, add it when we support 1.3 models